nx.json

```diff
@@ -17,51 +17,93 @@
     }
   },
   "namedInputs": {
-    "global": [
-      "{workspaceRoot}/yarn.lock",
+    "workspace": [
       "{workspaceRoot}/package.json",
+      "{workspaceRoot}/nx.json",
+      "{workspaceRoot}/.nxignore",
+      "{workspaceRoot}/tools/@o3r/workspace-helpers/scripts/*.mjs",
+      "{workspaceRoot}/jest.config.js",
+      "{workspaceRoot}/.github/workflows/*.yml"
```

Comment:
Why should we rebuild everything if we change something in the workflows ?

Reply:
Indeed, I followed Nx Cloud generator recommendation but I do not have good argument for it.